ungrib/Variable_Tables/Vtable.RAP.hybrid.ncep

```diff
@@ -9,15 +9,18 @@ Param| Type |Level1|Level2| Name     |  Units   | Description
   34 | 109  |   *  |      | VV       | m s-1    | V                                       |  0  |  2  |  3  | 105 |
  153 | 109  |   *  |      | QC       | kg kg-1  | Cloud water mixing ratio                |  0  |  1  | 22  | 105 |
  170 | 109  |   *  |      | QR       | kg kg-1  | Rain water mixing ratio                 |  0  |  1  | 24  | 105 |
-  58 | 109  |   *  |      | QI       | kg kg-1  | Ice mixing ratio                        |  0  |  6  |  0  | 105 |
+  58 | 109  |   *  |      | QI       | kg kg-1  | Ice mixing ratio                        |  0  |  1  | 82  | 105 |
  171 | 109  |   *  |      | QS       | kg kg-1  | Snow water mixing ratio                 |  0  |  1  | 25  | 105 |
  179 | 109  |   *  |      | QG       | kg kg-1  | Graupel mixing ratio                    |  0  |  1  | 32  | 105 |
- 148 | 107  |   *  |      | QNR      | kg-1     | Rain number concentration               |  0  |  3  | 195 | 105 |
- 198 | 109  |   *  |      | QNI      | kg-1     | Ice number concentration                |  0  |  1  | 207 | 105 |
+ 153 | 109  |   *  |      | QNR      | kg-1     | Rain number concentration               |  0  |  1  | 100 | 105 |
+ 255 | 109  |   *  |      | QNC      | kg-1     | Cloud number concentration              |  0  |  6  | 28  | 105 |
+ 198 | 109  |   *  |      | QNI      | kg-1     | Ice number concentration                |  0  |  6  | 29  | 105 |
+ 157 | 107  |   *  |      | QNWFA    | kg-1     | Water-fr. aerosol number concentration  |  0  | 13  | 193 | 105 |
+ 156 | 107  |   *  |      | QNIFA    | kg-1     | Ice-fr.   aerosol number concentration  |  0  | 13  | 192 | 105 |
      | 109  |   *  |      | RH       | %        | Relative Humidity                       |  0  |  1  |     | 105 |
   11 | 105  |   2  |      | TT       | K        | Temperature at 2 m                      |  0  |  0  |  0  | 103 |
   51 | 105  |   2  |      | SPECHUMD | kg kg-1  | Specific Humidity at 2 m                |  0  |  1  |  0  | 103 |
```

Comment:
@jimbresch @gthompsnWRF 
Greg,
Is there now a 3d RH field? If not, then other than having a surface RH laying around, we can't really use it.

Reply:
@davegill Wasn't my change for Vtable only the RH field at surface/2meters?  In which case the 3rd column of the GRIB2 parameters and first column of GRIB1 parameters was lacking a value and put those in.  I probably have to reopen my sample GRIB2 file to ensure RH was a field given at 2m.  I didn't intend to edit the line for 3D RH, only the surface.

Reply:
Greg,
The point was if we do _not_ have a 3d RH field, then I do not see any reason to add the surface RH into the Vtable. The metgrid program will just pass t his 2d array on through and the real program will ignore.
Sorry for being confusing in the morning.

Reply:
@davegill  Dave, actually I better have another look.  Because if RAP files have RH at 2m but not SPECHUMD, then we won't get an output RH at all for that level.  The hybrid level files are such a mixture of things, that I can no longer recall from memory and should do a g2print to find out for certain.  I now recall that a variable listed without the parameter number itself is a cause for calculating it downstream.  So that is how SPECHUMD gets used to calculate a RH field at the real.exe step.

Also, if my (vague) memory is correct, the newest RAP/HRRR files furthermore aligned the soil variables at 0cm to use the same vertical level coding as all the other levels, and you can note the existing Vtable has a different value for 0cm - because I think it was correct pre-July2018.  Like I said before, these changes occur without notice by most users, but doing something with the dates in comments is probably the only viable solution.

Reply:
Greg,
No rush. We will target this Vtable mod for v4.1.1.
That is more than 1 month away.